project-koku/masu
# 93 celery routing

* Remove the shared_task decorator in favor of assigning tasks specifically to the instantiated celery application. 
* Create a universal celery app--instead of using an app factory
  * Do application-specific config updates in a new update function

Manual testing: 

* The download API endpoint inserts an event into the `download` queue, as expected (instead of the default celery queue).
* The scheduled polling task is using the celery queue, and assigning tasks to the appropriate download, and process queues. 
* flower in OpenShift is happy
* download/process tasks run in OpenShift via the download API.
